### PR TITLE
Replace Monaco with IBM Plex

### PR DESCRIPTION
**Pull request by robinrendle**
### In this PR

- Switching out Monaco with IBM Plex for code font
- Fixes faux bold weight problem 
- IBM Plex rules

---

### Example: Tags

Before:

Screen Shot 2020-10-09 at 11 58 35 AM

After:

Screen Shot 2020-10-09 at 11 58 59 AM

---

### Example: Breadcrumbs

Before:

Screen Shot 2020-10-09 at 11 38 47 AM

After:

Screen Shot 2020-10-09 at 11 59 57 AM

### CODE REVIEW

**Review by ckj — APPROVED** [review]

**Comment by joshuarli** [comment]
Damn, this looks nice.

File: tests/acceptance/test_issue_details.py [hunk 1]
```diff
@@ -105,7 +105,7 @@ def test_javascript_specific_event(self):
         self.page.visit_issue(self.org.slug, event.group.id)
         self.browser.snapshot("issue details javascript - event details")
 
-        self.browser.find_element_by_xpath("//button//span[contains(text(), 'curl')]").click()
+        self.browser.click('[data-test-id="curl"]')
```
Comment (billyvg) [inline tests/acceptance/test_issue_details.py:108]: We shouldn't need a `data-test-id` for Button, you can use `aria-label`

```suggestion
        self.browser.click('[aria-label="curl"]')
```

Comment (robinrendle) [reply tests/acceptance/test_issue_details.py:108]: I just checked this and it wasn’t adding an aria-label to the button until I added the text translation thingy?

Comment (robinrendle) [reply tests/acceptance/test_issue_details.py:108]: So since we don’t want `curl` to be translated, I added the data-test-id instead

Comment (robinrendle) [reply tests/acceptance/test_issue_details.py:108]: Tell me if I’m being a pillock here though

Comment (billyvg) [reply tests/acceptance/test_issue_details.py:108]: @robinrendle Hmm, it should add an `aria-label` w/o the translation - I think `<Button>` has the same issue with AlertLink where it does not pass the `data-test-id` prop down.

File: tests/acceptance/test_issue_details.py [hunk 2]
```diff
@@ -51,118 +51,22 @@ def create_sample_event(self, platform, default=None, sample_name=None, time=Non
...
-
-    def test_android_ndk_event(self):
-        event = self.create_sample_event(default="android-ndk", platform="android-ndk")
-        self.page.visit_issue(self.org.slug, event.group.id)
-        self.browser.snapshot("issue details android-ndk")
-
-    def test_aspnetcore_event(self):
-        event = self.create_sample_event(default="aspnetcore", platform="csharp")
-        self.page.visit_issue(self.org.slug, event.group.id)
-        self.browser.snapshot("issue details aspnetcore")
```
Comment (markstory) [inline tests/acceptance/test_issue_details.py:101]: Did you mean to remove all these tests?

File: src/sentry/static/sentry/app/components/events/interfaces/breadcrumbs/index.tsx [hunk 3]
```diff
@@ -372,15 +373,17 @@ class Breadcrumbs extends React.Component<Props, State> {
...
-          <List
-            breadcrumbs={filteredBySearch}
-            event={event}
-            orgId={orgId}
-            onSwitchTimeFormat={this.handleSwitchTimeFormat}
-            displayRelativeTime={displayRelativeTime}
-            searchTerm={searchTerm}
-            relativeTime={relativeTime!} // relativeTime has to be always available, as the last item timestamp is the event created time
-          />
+          <ErrorBoundary>
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/events/interfaces/breadcrumbs/index.tsx:376]: cc @priscilawebdev

Comment (priscilawebdev) [reply src/sentry/static/sentry/app/components/events/interfaces/breadcrumbs/index.tsx:376]: oh thanks for pointing it out...let me check it

**Pull request merged**